lang/jvm/src/main/scala/com/wavesplatform/lang/Global.scala

```diff
@@ -38,7 +38,7 @@ object Global extends BaseGlobal {
     } yield result
 
   override def base16EncodeImpl(input: Array[Byte]): Either[String, String] =
-    toEither(BaseEncoding.base16().encode(input))
+    toEither(BaseEncoding.base16().encode(input)).map(_.toLowerCase)
```

Comment:
I suggest caching a pre-configured instance of an encoding instead of calling `toLowerCase`:
```
val encoding = BaseEnconding.base16().lowerCase()
...
Right(encoding.encode(input))
```

Reply:
ok